pyproject.toml

```diff
@@ -138,23 +135,48 @@ lxml-stubs = "^0.5.1"
 django-debug-toolbar = "^5.1.0"
 django-tailwind = {extras = ["reload"], version = "^3.8.0"}
 
-[tool.isort]
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-use_parentheses = true
-ensure_newline_before_comments = true
-line_length = 79
-skip_glob = "*/migrations/*.py"
-
-[tool.pylint.messages_control]
-disable = "C0330, C0326"
-
-[tool.pylint.format]
-max-line-length = "79"
-
 [tool.ruff]
 line-length = 79
+lint.extend-safe-fixes = [
+  # non-pep585-annotation
+  "UP006",
+]
+lint.select = [
+  # pycodestyle
+  "E",
+  # Pyflakes errors
+  "F",
+  # isort
+  "I",
+  # pyupgrade
+  "UP",
+  # Pyflakes warnings
+  "W",
+]
+lint.ignore = [
+  # flake8-bugbear opinionated rules
+  "B9",
+  # line-too-long
+  "E501",
+  # suppressible-exception
+  "SIM105",
+  # if-else-block-instead-of-if-exp
+  "SIM108",
+  # printf-string-formatting
+  "UP031",
+  # unnecessary-collection-call
+  "C408",
+  # To fix:
+  "E402",
+  "E721",
+  "E722",
+  "E731",
+  "F403",
+  "F811",
+  "F821",
+  "F841",
+  "UP008",
```

Comment:
To reduce the churn and risk, I've ignored a bunch of extra rules in this repository. Given time, I hope to come back and fix them in small batches.